cross-modal-search/README.md

```diff
@@ -173,6 +129,52 @@ Because the user does not want to see the vector as a result, but the image this
 Note, that this Flow only shows how to search for images using text. The example actually support searching for text using images as well. 
 As an exercise, you can think of the required steps for that and check against our [Flow configuration](flows/flow-query.yml).
 
+
+## 📖 Optional: Extra information useful for the user
+<details>
```

Comment:
We don't need to hide it here. This section is specially for that

Reply:
Ture, b7138b060992d15e287947fe34c829163415cec8